alerting/provider/gitlab/gitlab.go

```diff
@@ -0,0 +1,147 @@
+package gitlab
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/TwiN/gatus/v5/alerting/alert"
+	"github.com/TwiN/gatus/v5/client"
+	"github.com/TwiN/gatus/v5/core"
+	"github.com/google/uuid"
+)
+
+// AlertProvider is the configuration necessary for sending an alert using GitLab
+type AlertProvider struct {
+	WebhookURL       string `yaml:"webhook-url"`       // The URL of the GitHub repository to create issues in
+	AuthorizationKey string `yaml:"authorization-key"` // Token requires at least RW on issues and RO on metadata
```

Comment:
Shouldn't these two be `repository-url` and `token` respectively? Googling `gitlab authorization key` doesn't yield any concise results.

Or is the former actually a webhook?

Reply:
![image](https://github.com/TwiN/gatus/assets/193949/b2a28b73-56dd-4096-b13f-62f840b43967)
It's called that in the GitLab UI